src/main/java/DukeException.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Encapsulates exceptions that are for the Duke Task manager.
+ * It handles user's input error. The class provides extenstion to other Duke specific exception classes.
+ */
+public class DukeException extends Exception {
+
+    /**
+     * Constructs a DukeException object.
+     *
+     * @param message This contains the message of the error.
+     *                The message of the error will be shown to the user when thrown.
+     */
+    public DukeException(String message) {
+        super(message);
+    }
+}
+
+/**
+ * Encapsulates exception that will be thrown when Todo description is empty.
+ */
+class EmptyToDoDescriptionException extends DukeException {
```

Comment:
Why does only the `ToDo` class have its own exception, and not `Deadline` or `Event`?